index.html

```diff
@@ -18,10 +18,37 @@
 <body>
   <header>
     <div class="container">
+      <h1>Code Journal</h1>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <form action="">
+        <h1>New Entry</h1>
+        <div class="col-md">
```

Comment:
`row` would be a more appropriate class name. This is technically a row, not a column. Your columns go inside a row.